frontend/whiteboard-app/src/index.css

```diff
@@ -11,3 +11,19 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/*add button css
```

Comment:
Wir entfernen auskommentierten Code immer, sonst ist der main branch irgendwann voll damit.
Kommentare, die zum Verständnis vom Code da sind können gerne rein, aber Code in Kommentaren bitte raus.